OOStepByStep/Class.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOStepByStep
+{
+    public class Class
+    {
+        private Person person;
+
+        public Class(Person person)
+        {
+            this.person = person;
+        }
+
+        public string SelfIntroduceWithClassId(Person person, int classId)
```

Comment:
this behavior should belong to teacher or student
e.g.  teacher.SelfIntroduce(), student.SelfIntroduce()